Dockerfile

```diff
@@ -50,13 +51,8 @@ RUN apt-get update && \
 # Copy artifacts from build stage
 COPY --from=build /usr/local/bin/supertuxkart /usr/local/bin
 COPY --from=build /usr/local/share/supertuxkart /usr/local/share/supertuxkart
+COPY docker-entrypoint.sh /docker-entrypoint.sh
```

Comment:
This needs to be set as the entry point of the container via `ENTRYPOINT` at the end of the Dockerfile, otherwise the container will terminate immediately.

Reply:
Please ensure that the `docker-entrypoint.sh` is executable (`chmod +x docker-entrypoint.sh`). This does not seem to be the case at the moment.